### PR TITLE
test(mark as unread): latest post should appear unread after marking the channel as unread in LHS via menu option or shortcut key

### PR DESCRIPTION
**Pull request by jgilliam17**
#### Summary

Adding new test cases for mark channel as unread
#### Ticket Link

<a href="https://gitpod.io/#https://github.com/mattermost/mattermost-test-management/pull/5"></a>

### CODE REVIEW

File: data/test-cases/channels/mark-as-unread/mark-channel-unread1.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+1. Channel A starts as unread
+2. Visit Channel A
+    1. Verify channel A becomes read after you viewed it
+3. Switch to channel B
+    1. Verify Channel A remains read
+4. Mark channel A with the RHS mark as unread option
+    1. Verify channel A shows as unread on the sidebar
+5. Switch back to channel A
+    1. Verify the New Messages line should appear above the last post (post3)
```
Comment (saturninoabril) [inline data/test-cases/channels/mark-as-unread/mark-channel-unread1.md:49]: In preview mode, the indented statement for verify is showing either `1.` in editor or `i.` in GitHub (see screenshot).
Screen Shot 2022-11-05 at 5 58 20 AM

How about using the bullet point (either below)?

Screen Shot 2022-11-05 at 5 59 37 AM

Screen Shot 2022-11-05 at 5 59 17 AM

File: data/test-cases/channels/mark-as-unread/mark-channel-unread1.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+**Expected**
+
+When marking the channels as unread, the channel should show as unread on the sidebar. When opening a channel that was marked as unread manuually, the New Messages line should appear above the last post in the channel
```
Comment (saturninoabril) [inline data/test-cases/channels/mark-as-unread/mark-channel-unread1.md:53]:
```suggestion
When marking the channels as unread, the channel should show as unread on the sidebar. When opening a channel that was marked as unread manually, the New Messages line should appear above the last post in the channel
```

File: data/test-cases/channels/mark-as-unread/mark-channel-unread2.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
+---
+# (Required) Ensure all values are filled up
+name: "The latest post should appear unread after marking the channel as unread with alt-left-click on channel sidebar item"
```
Comment (saturninoabril) [inline data/test-cases/channels/mark-as-unread/mark-channel-unread2.md:3]: Please indicate the key when using Mac. It would be `option` key, right?

**Review by saturninoabril — CHANGES_REQUESTED** [review]
Thanks @jgilliam17! Really appreciate for you going ahead with this new process! Please see comments.

For other reviewer, I believe this is still not on master -- on going review at https://github.com/mattermost/mattermost-webapp/pull/10755. You may use the test server on that PR.

**Comment by saturninoabril** [comment]
Also, we're following conventional commits - https://www.conventionalcommits.org/en/v1.0.0/. Since we're doing "squash and merge", the commit message takes the PR title. With that, I'm recommending the following format for test cases PR:
```
test(feature): general description of tests
```

And so, please change to something like:
```
test(mark as unread): latest post should appear unread after marking the channel as unread in RHS via menu option or shortcut key
```

File: data/test-cases/channels/mark-as-unread/mark-channel-unread2.md (hunk 3 above)
Comment (jgilliam17) [reply data/test-cases/channels/mark-as-unread/mark-channel-unread2.md]: Great point. Fixed.

**Comment by jgilliam17** [comment]
Thanks @saturninoabril 
I've made all the changes as requested. Please take another look 🙂

**Comment by saturninoabril** [comment]
Thanks @jgilliam17! LGTM.

**Review by saturninoabril — APPROVED** [review]

**Comment by saturninoabril** [comment]
I'm merging this now.

**Pull request merged**